### PR TITLE
Using a single handler signal insead of before and after signals.

### PR DESCRIPTION
**Pull request by arteymix**
request_handler is splitted in handler signal and soup_handler method.

It is simpler to have a signal handling the request on which you can
connect and connect_after than having 2 distincts signals.

### CODE REVIEW

File: src/router.vala [hunk 1]
```diff
@@ -30,14 +19,23 @@ namespace Valum {
 
 		public Router() {
 
+			this.handler.connect((req, res) => {
+				res.status = 200;
+				res.mime   = "text/html";
+			});
+
+			this.handler.connect_after((req, res) => {
+			res.message.response_body.complete ();
```
Comment (antono) [inline src/router.vala]: Looks like tab missing.

Comment (arteymix) [inline src/router.vala]: Le Thursday 08 January 2015 à 00:50 -0800, Antono Vasiljev a écrit :
> In src/router.vala:
> 
> > @@ -30,14 +19,23 @@ namespace Valum {
> >  
> >  		public Router() {
> >  
> > +			this.handler.connect((req, res) => {
> > +				res.status = 200;
> > +				res.mime   = "text/html";
> > +			});
> > +
> > +			this.handler.connect_after((req, res) => {
> > +			res.message.response_body.complete ();
> 
> Looks like tab missing.
> 
> —
> Reply to this email directly or view it on GitHub.
> 
> 

Fixed!

-- 
Guillaume Poirier-Morency <guillaumepoiriermorency@gmail.com>

**Comment by arteymix** [comment]
Okay, I fixed the PR.

**Comment by antono** [comment]
Thank you!

**Comment by arteymix** [comment]
It's a pleasure!

**Pull request merged**